### ISSUE TITLE
learning rate - when big batch size

### ISSUE THREAD

**Issue by MM-0712** [wenet-e2e/wespeaker#388]
Hi，
Has the community done any ablation experiments on batch size and learning rate?
In my experiments, when the batch size increases from 256 to 768 (2 gpus, the same dataset), the EER will be worsen (6.65 -> 7.38) if the learning rate policy remains unchanged.

**Comment by JiJiJiang** [comment]
No, we did not do the ablation experiments.
You could try training more iterations by using a bigger epoch_num.
